addons/point_of_sale/static/src/js/Popups/AlertPopup.js

```diff
@@ -0,0 +1,26 @@
+odoo.define('point_of_sale.AlertPopup', function(require) {
+    'use strict';
+
+    const AbstractAwaitablePopup = require('point_of_sale.AbstractAwaitablePopup');
+    const Registries = require('point_of_sale.Registries');
+    const { _lt } = require('@web/core/l10n/translation');
+
+    // formerly AlertPopupWidget
+    class AlertPopup extends AbstractAwaitablePopup {
+        setup() {
+            super.setup();
+        }
+    }
```

Comment:
The super call to setup is not needed. You can just do:
class AlertPopup extends AbstractAwaitablePopup { }
Just like in the confirmPopup?